gp_emulator/gpu/kernel_crossMinus.cu

```
@@ -0,0 +1,35 @@
+#include "gpu_predict.h"
+/*********************************************//**
+ * cross minus:
+ * aa_{ix, iy} = inputs_{ix} - testing_{iy}
+ *********************************************/
+__global__
+void kernel_crossMinus(const real *v1,const real *v2, real *mat_res, const int v1_len, const int v2_len)
+{
+    int ix = blockIdx.x * blockDim.x + threadIdx.x;
+    int iy = blockIdx.y * blockDim.y + threadIdx.y;
+
+    if(ix < v1_len && iy < v2_len)
```

Do you need to raise an exception if this condition is not satisfied? Explain...